README.md

```diff
@@ -1 +1,13 @@
-# vagrant-is
\ No newline at end of file
+# WSO2 Identity Server Vagrant Resources
+
+This repository contains the following resources:
+
+- Vagrant box generation
+
+Resources for creating Vagrant boxes are contained here. Users can define box specifications
+based on which the Vagrant boxes are generated.
+
+- Vagrantfile for WSO2 Identity Server deployment with Analytics
```

Comment:
Can be better named as "Resources to deploy WSO2 Identity Server with Analytics on Vagrant""

Reply:
+1